openedx/core/djangoapps/catalog/tests/test_utils.py

```diff
@@ -58,27 +58,46 @@ def assert_contract(self, call_args, program_uuid=None, types=None):  # pylint:
 
         self.assertEqual(kwargs['api']._store['base_url'], self.catalog_integration.internal_api_url)  # pylint: disable=protected-access
 
-        querystring = {
-            'marketable': 1,
-            'exclude_utm': 1,
-        }
-        if program_uuid:
-            querystring['use_full_course_serializer'] = 1
-        if types:
-            querystring['types'] = types_param
+        if expected_querystring:
+            querystring = expected_querystring
+        else:
+            querystring = {
```

Comment:
We should also remove this test case after https://github.com/edx/course-discovery/pull/805  is merged right? Can you add a todo here as well?